src/scaffoldmaker/meshtypes/meshtype_3d_bladderurethra1.py

```diff
@@ -0,0 +1,923 @@
+                'Use cross derivatives': False,
+                'Refine': False,
+                'Refine number of elements around': 4,
+                'Refine number of elements along': 4,
+                'Refine number of elements through wall': 1
+            },
+        }),
+        'Ostium Rat 1': ScaffoldPackage(MeshType_3d_ostium1, {
+            'scaffoldSettings': {
+                'Number of vessels': 1,
+                'Number of elements across common': 2,
+                'Number of elements around ostium': 8,  # implemented for 8
+                'Number of elements along': 1,
+                'Unit scale': 1.0,
+                'Outlet': False,
+                'Ostium diameter': 1.0,
+                'Ostium length': 0.5,
+                'Ostium wall thickness': 0.02,
+                'Use linear through ostium wall': True,
+                'Vessel inner diameter': 0.4,
+                'Vessel wall thickness': 0.1,
+                'Vessel angle 1 degrees': 0.0,
+                'Vessel angle 1 spread degrees': 0.0,
+                'Vessel angle 2 degrees': 0.0,
+                'Use linear through vessel wall': True,
+                'Use cross derivatives': False,
+                'Refine': False,
+                'Refine number of elements around': 4,
+                'Refine number of elements along': 4,
+                'Refine number of elements through wall': 1
+            },
+        })
+    }
+
+    @staticmethod
+    def getName():
+        return '3D Bladder Urethra 1'
+
+    @staticmethod
+    def getParameterSetNames():
+        return [
+            'Default',
+            'Cat 1',
+            'Rat 1']
+
+    @classmethod
+    def getDefaultOptions(cls, parameterSetName='Default'):
+        if 'Rat 1' in parameterSetName:
+            centralPathOption_LUT = cls.centralPathDefaultScaffoldPackages_LUT['Rat 1']
+            centralPathOption_bladder = cls.centralPathDefaultScaffoldPackages_bladder['Rat 1']
+        else:
+            centralPathOption_LUT = cls.centralPathDefaultScaffoldPackages_LUT['Cat 1']
+            centralPathOption_bladder = cls.centralPathDefaultScaffoldPackages_bladder['Cat 1']
+        if 'Rat 1' in parameterSetName:
+            ostiumOption = cls.ostiumDefaultScaffoldPackages['Ostium Rat 1']
+        else:
+            ostiumOption = cls.ostiumDefaultScaffoldPackages['Ostium Cat 1']
+        options = {
+            'Central path LUT': copy.deepcopy(centralPathOption_LUT),
+            'Central path bladder': copy.deepcopy(centralPathOption_bladder),
+            'Number of elements along': 20,
+            'Number of elements around': 12,
+            'Number of elements through wall': 1,
+            'Major diameter': 30.0,
+            'Minor diameter': 25.0,
+            'Neck diameter 1': 5.0,
+            'Neck diameter 2': 3.0,
+            'Bladder wall thickness': 0.5,
+            'Neck angle': 45,
+            'Include urethra': True,
```

Comment:
If we have annotation groups on the central path, then making the urethra group empty could indicate remove the urethra?

Reply:
I'm happy with 'Include urethra', easy to use.